docker/linux/unittest/Dockerfile

```diff
@@ -33,6 +37,15 @@ RUN apt update -qqy \
         unzip \
         xauth \
         xvfb \
+        libasound2-dbgsym \
+        # libavcodec58-dbgsym \
+        # libavformat58-dbgsym \
```

Comment:
Why are these two commented out?

Reply:
Removed. I couldn't figure out how to get the correct sources.list upstream for this ( yet ). There's some version mismatch.

But those are also not critical for what i need right now.